eol/alpine/eof.go

```diff
@@ -0,0 +1,109 @@
+package alpine
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/PuerkitoBio/goquery"
+	"github.com/aquasecurity/vuln-list-update/utils"
+	"github.com/spf13/afero"
+	"golang.org/x/xerrors"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+)
+
+const (
+	eolAlpineFolder = "eol/alpine"
+	eolAlpineFile   = "eol-alpine"
+	eolAlpineUrl    = "https://alpinelinux.org/releases/"
+	retry           = 5
+)
+
+type Config struct {
+	vulnListDir string
+	eolUrl      string
+	retry       int
+	appFs       afero.Fs
+}
+
+func NewConfig() Config {
+	return Config{
+		vulnListDir: utils.VulnListDir(),
+		eolUrl:      eolAlpineUrl,
+		retry:       retry,
+		appFs:       afero.NewOsFs(),
+	}
+}
+
+func (config Config) Update() error {
```

Comment:
nit: a single letter name is fine.

```suggestion
func (c Config) Update() error {
```

Reply:
Changed